Colors/Pallet/ImageProcessor.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PalletLib
+{
+    public class ImageProcessor
+    {
+        public static Bitmap GetBitmap(string path)
+        {
+            if (File.Exists(path))
+            {
+                try
+                {
+                    using (FileStream fs = File.OpenRead(path))
+                    {
+                        return new Bitmap(fs);
+                    }
+
+                    // return new Bitmap(path); // Alternative way
+                }
+                catch
+                {
+                    throw new Exception("Not a bitmap image!!!");
+                }
+            }
+            else
+            {
+                throw new FileNotFoundException($"File {path} does not exist!!!");
+            }
+        }
+
+        public static Pallet GetPallet(string path)
+        {
+            var bitmap = GetBitmap(path);
```

Comment:
try create Bitmap from path in constructor and save it to the class field on class level - then you can use it everywhere in methods (and remove static from them) without creating "new Bitmap" every time